cmd/restic/exclude.go

```diff
@@ -385,6 +387,28 @@ func rejectBySize(maxSizeStr string) (RejectFunc, error) {
 	}, nil
 }
 
+// rejectCloudFiles returns a func which rejects files which are online-only cloud files
+func rejectCloudFiles() (RejectFunc, error) {
+	if runtime.GOOS != "windows" {
+		return nil, errors.Fatalf("exclude-cloud-files is only supported on Windows")
+	}
+
+	const (
+		FILE_ATTRIBUTE_UNPINNED              = 0x00100000
+		FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS = 0x00400000
```

Comment:
Unfortunately these are [not defined in syscall](https://pkg.go.dev/syscall?GOOS=windows#pkg-constants), probably due to their uncommon usage.

They are documented by Microsoft here [File Attribute Constants (WinNT.h)](https://learn.microsoft.com/en-us/windows/win32/fileio/file-attribute-constants).

Reply:
The constants are available in https://pkg.go.dev/golang.org/x/sys/windows .